build.sh

```diff
@@ -87,34 +68,25 @@ if hasArg clean; then
     # The find removes all contents but leaves the dirs, the rmdir
     # attempts to remove the dirs but can fail safely.
     for bd in ${BUILD_DIRS}; do
-    if [ -d ${bd} ]; then
-        find ${bd} -mindepth 1 -delete
-        rmdir ${bd} || true
+    if [ -d "${bd}" ]; then
+        find "${bd}" -mindepth 1 -delete
+        rmdir "${bd}" || true
     fi
     done
 fi
 
-if (( ${BUILD_ALL_GPU_ARCH} == 0 )); then
-    GPU_ARCH="-DGPU_ARCHS="
-    echo "Building for the architecture of the GPU in the system..."
-else
-    GPU_ARCH="-DGPU_ARCHS=ALL"
-    echo "Building for *ALL* supported GPU architectures..."
-fi
-
```

Comment:
We don't export or pass `GPU_ARCH` to anything, and for a pure-python package I don't think it's doing anything